ansible/roles/benchmarking/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+
+- name: Install h2load dependencies
+  package:
+    name: "{{ package }}"
+    state: present
+  loop_control:
+    loop_var: package
+  with_items:
+    - libc-ares-dev
+    - libcunit1-dev
+    - libev-dev
+    - libevent-dev
+    - libjansson-dev
+    - libssl-dev
+    - libxml2-dev
+    - pkg-config
+    - python-eventlet
+    - python-numpy
+    - python-requests
+    - zlib1g-dev
+  become: true
+  become_user: iojs
+
+- name: Download h2load git repo
+  git:
+    repo: 'https://github.com/nghttp2/nghttp2.git'
+    dest: "/home/{{ server_user }}/nghttp2"
+  become: true
+  become_user: iojs
+
+- name: Setup and compile h2load
+  shell: "autoreconf -i && automake && autoconf && ./configure --enable-app && make -j4"
+  args:
+    chdir: "/home/{{ server_user }}/nghttp2"
+  become: true
+  become_user: iojs
+
+- name: Add h2load to $PATH
+  lineinfile:
+    path: "/home/{{ server_user }}/.bashrc"
+    regexp: '^export PATH='
+    line: "export PATH=$PATH:/home/{{ server_user }}/nghttp2/src"
+  become: true
+  become_user: iojs
```

Comment:
I'm not sure if this will achieve what is desired as I'm now 100% sure the .bashrc will run when the jenkins agent is started.

Usually, paths have been added in the file used to start the jenkins agent.  For example: https://github.com/nodejs/build/blob/master/ansible/roles/jenkins-worker/templates/upstart.j2